pkgs/development/compilers/llvm/15/clang/default.nix

```diff
@@ -60,6 +62,10 @@ let
 
     outputs = [ "out" "lib" "dev" "python" ];
 
+    env = lib.optionalAttrs (stdenv.buildPlatform != stdenv.hostPlatform) {
+      NIX_CFLAGS_COMPILE = "-Wno-maybe-uninitialized";
+    };
```

Comment:
It'd be nice to narrow this to the `gcc` versions that trigger the error but not worth revisiting if you don't have that information handy.

Could you leave a note mentioning what you said in your comment (lint triggers under recent gcc for cross builds only) so that we know to revisit this when updating LLVM?

Reply:
40f9e80d505f7d866bb6b2035832681a0ed2c3d8